include/aie/Dialect/AIEX/IR/AIEX.td

```diff
@@ -791,6 +791,27 @@ def AIE_NpuAddressPatchOp: AIEX_Op<"npu.address_patch", []> {
   }];
 }
 
+def AIE_NpuControlPacketOp: AIEX_Op<"npu.control_packet", []> {
```

Comment:
`npu` prefix is not part of the original PR. This is not an npu op. Control packets are part of AIE, so should be `aie.control_packet` (for now in `aiex`)

Reply:
Great point. Npu prefix is removed. Makes sense that this feature should eventually be made available for AIE1 too.